universal-application-tool-0.0.1/app/assets/javascripts/azure_delete.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This is responsible for deleting a file that was uploaded to Azure blob storage.
+ */
+
+class AzureDeleteController {
+  constructor() {
+    window.addEventListener('delete', (event) => this.attemptDelete())
+  }
+
+  attemptDelete() {
+    const azblob = window['azblob']
+    const blockBlobUrl = this.getBlockBlobUrl()
+    if (blockBlobUrl == null) {
```

Comment:
maybe consider doing !blockBlobUrl here instead of == if you want to catch anything besides null or undefined values